README.md

```diff
@@ -1,246 +1,211 @@
+<img src="docs/pics/logo/KMESH-horizontal-colour.png" alt="kmesh-logo" style="zoom: 100%;" />
+
+## Introduction
+
+Kmesh is a high-performance service mesh data plane software based on programmable kernel. Provides high-performance service communication infrastructure in service mesh scenarios.
+
+## Why Kmesh
+
+### Challenges of the Service Mesh Data Plane
+
+The service mesh software represented by Istio has gradually become popular and has become an important component of cloud infrastructure. However, the current service mesh still face some challenges:
+
+- **Extra latency overhead at the proxy layer**: Single hop service access increases by [2~3ms](https://istio.io/latest/docs/ops/deployment/performance-and-scalability/#data-plane-performance), which cannot meet the SLA requirements of latency-sensitive applications. Although the community has come up with a variety of data plane solutions to this problem, the overhead introduced by agents cannot be completely reduced.
+- **High resources occupation**: The sidecar proxy occupies extra CPU/MEM overhead, and the deployment density of service container decreases.
+
+### Kmesh：Kernel-native traffic governance
+
+Kmesh innovatively proposes to move traffic governance to the OS, and build a transparent sidecarless service mesh without passing through the proxy layer on the data path.
+
+![image-20230927012356836](docs/pics/why-kmesh-arch-en.png)
+
+### Key features of Kmesh
+
+**Smooth Compatibility**
+
+- Application-transparent Traffic Management
+- Automatically interconnecting with Istiod
+
+**High Performance**
+
+- Forwarding delay **60%↓**
+- Service startup performance **40%↑**
+
+**Low Overhead**
+
+- ServiceMesh data plane overhead **70%↓**
+
+**Safety Isolation**
+
+- eBPF Virtual machine security
+- Cgroup level orchestration isolation
+
+**Full Stack Visualization**
+
+- E2E observation*
+- Integration with Mainstream Observability Platforms*
+
+**Open Ecology**
+
+- Supports XDS protocol standards
+
+Note: * Planning
+
+## Quick Start
+
+- Prerequisite
+
+  - Currently, Kmesh makes use of [Istio](https://github.com/istio/istio) as its control plane. Before starting Kmesh, please install the Istio control plane. To support ads mode, refer to [istio install guide](https://istio.io/latest/docs/setup/getting-started/#install), but to support workload mode, please refer to [ambient install guide](https://istio.io/latest/docs/ops/ambient/install/helm-installation/#installing-the-components).
+  
+  - The complete Kmesh capability depends on the OS enhancement. Check for the [OS lists](docs/kmesh_support.md) supported by Kmesh. For other OS environments, refer to [Kmesh Compilation and Building Guide](docs/kmesh_compile.md) to build kesh image locally.
+
+- Kmesh Install
+
+  Install kmesh with helm
+  
+  ```sh
+  [root@ ~]# helm install kmesh ./deploy/helm -n kmesh-system --create-namespace
```

Comment:
you'd better setup a chart repo or use oci image, that would be help to avoid to clone the repo.

Reply:
Absolutely right